modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -85,43 +86,35 @@ spec:
         key: credentials
         name: cloud-credentials
       default: true
+      config:
+        region: ${REGION}
   configuration:
     velero:
       defaultPlugins:
       - openshift
       - aws
       restic:
         enable: false
-  volumeSnapshots:
+  snapshotLocations:
   - velero:
       config:
-        credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials
-        enableSharedConfig: "true"
-        region: ${REGION}
+        credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials <1>
+        enableSharedConfig: "true" <2>
+        profile: default <3>
+        region: ${REGION} <4>
       provider: aws
 EOF
 ----
+
++
+<1> The `credentialsFile` is the mounted location of the bucket credential on the pod.
+<2> The `enableSharedConfig` allows the `snapshotLocations` to share or reuse the credential defined for the bucket.
+<3> Assume your Velero default for your `profile: default`. 
+<4> Specify `region` as your AWS region. This must be the same as the cluster region.
 +
 [NOTE]
 ====
 The `enable` parameter of `restic` is set to `false` in this configuration because OADP does not support Restic in ROSA environments.
 ====
 +
 You are now ready to backup and restore OpenShift applications, as described in the link:https://docs.openshift.com/container-platform/4.11/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html[OADP documentation].
```

Comment:
I onow this is existing text, should this be a `Next step`?

Reply:
@weshayutin provided this info as correct and not a next step. thanks